nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -72,15 +73,103 @@ in {
           https://github.com/PipeWire/pipewire/blob/master/NEWS
         '';
       };
+
+      config = {
+        pipewire = mkOption {
+          type = lib.types.attrsOf json.type;
+          default = {};
+          example = {
+            "10-clock-rate" = {
+              "context.properties" = {
+                "default.clock.rate" = 44100;
+              };
+            };
+            "11-channelmix-upmix" = {
+              "stream.properties" = {
+                "channelmix.upmix" = false;
+              };
+            };
+          };
+          description = lib.mdDoc ''
+            Configuration for PipeWire. For details see
```

Comment:
Can you elaborate on the docs a little bit here? Also, the links can be made Markdown.

Reply:
Improved the docs and converted the links to markdown